ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -637,8 +643,6 @@ export default defineComponent({
 
         if (!mhrTransferFiling.error) {
           setUnsavedChanges(false)
-          // Delete the draft on successful submission
-          if (getMhrInformation.value.draftNumber) await deleteMhrDraft(getMhrInformation.value.draftNumber)
```

Comment:
The API requires the `draftNumber` in the payload in order to DELETE the draft. 
We previously didn't include said draftNumber, so this was done in error. 

Now that we are including the draftNumber and following the spec, we no longer require this 👍

Reply:
Thanks for catching this.

Reply:
No prob at all, it was my mistake, so easy to find 😆